tools/templates/vfvs_run.py

```diff
@@ -767,7 +1246,6 @@ def process(ctx):
 
     for collection_key in collections:
         collection = collections[collection_key]
-        collection_full_name = collection['collection_full_name']
```

Comment:
Should merge in the current code (this is backing out a recent change)

Reply:
I wanted to confirm something here. The code block that uses this code is: 
 ```
       collection_full_name = collection['collection_full_name']
        collection_count = collection['ligand_count']
        preprocess_collection(ctx, collection)
        collections[collection_full_name] = collection
```
If I remove the line: 
`        collection_full_name = collection['collection_full_name']
`
The subsequent line  (that uses variable collection_full_name) will give a bug?